org.eclipse.m2e.core.ui/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.m2e.core.ui;singleton:=true
-Bundle-Version: 1.16.0.qualifier
+Bundle-Version: 1.17.0.qualifier
```

Comment:
As this change doesn't introduce new APIs, you could just stick to 1.16.1.qualifier, and that would require less changes in consumers.

Reply:
> As this change doesn't introduce new APIs, you could just stick to 1.16.1.qualifier, and that would require less changes in consumers.

The extension point is part of the UI bundle, wouldn't that be considered API?

Reply:
In practice, extension points are all considered as API unless the documentation states otherwise. I don't think the plugin that hosts it has an infuence.

Reply:
Sorry, I misunderstood. Yes, you're right that extension point is API, so 1.17.0 is correct.